emctl/cmd/client/command/meshinstall/shadowservice/deploy.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package shadowservice
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/megaease/easemeshctl/cmd/client/command/flags"
+	installbase "github.com/megaease/easemeshctl/cmd/client/command/meshinstall/base"
+	"github.com/pkg/errors"
```

Comment:
the order of import package need to be adjusted.

Reply:
This order is managed by the gofmt tools..., I'm using the default configuration of vscode, we need to use the same gofmt configuration to keep it stable.

And, I think the order is correct as `megaease` is before `pkg` in alphabet order.

Reply:
Just putting the "github.com/pkg/errors" to another group is OK [1]

[1] https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Do not mix it all together, add a blank line between our own library and the third part library.

Reply:
Thanks, updated according to the comment.